packages/components/src/components/FileUploader/useFileUploader.ts

```diff
@@ -26,8 +26,12 @@ import {
 
 export type FileUploaderHookProps = {
   id: string | undefined;
-  /**Dersom komponenten styres utenfra. En liste over filer som skal være med ved første render. */
+  /**En liste over filer som skal være med ved første render. */
   initialFiles: FileList | undefined;
+  /**Dersom komponenten styres utenfra. En liste over filer som er med i hver render. */
```

Comment:
Som konsument så tror jeg ikke denne beskrivelsen blir særlig beskrivende for hva slags filer som skal være her 🤔 Kanskje noe som dette:
```suggestion
  /**Dersom komponenten styres utenfra. Filene som har blitt lastet opp. */
```